projects/organization-management/src/app/store/users/users.actions.ts

```diff
@@ -23,4 +28,31 @@ export class LoadUsersSuccess implements Action {
   constructor(public payload: { users: User[] }) {}
 }
 
-export type UsersAction = LoadUsers | LoadUsersFail | LoadUsersSuccess;
+export class LoadUsersRoles implements Action {
+  readonly type = UsersActionTypes.LoadUsersRoles;
+  constructor(public payload: { users: User[] }) {}
+}
+
+export class LoadUserRoles implements Action {
+  readonly type = UsersActionTypes.LoadUserRoles;
+  constructor(public payload: { user: User }) {}
+}
```

Comment:
For the request, only the user email is used, right? So the payload should only contain the email, not the whole user object.